content/millennium/dstu2/medications/medication-order.md

```diff
@@ -79,8 +79,17 @@ _Implementation Notes_
  `patient`                 | Y         | [`reference`] | The identity of a patient to list orders for. Example: `12345`
  `status`                  | N         | [`token`]     | The status of the medication order, may be a list separated by commas. Example: `active,draft`
  `-timing-boundsPeriod`    | N         | [`date`]      | The date-time which should fall within the [period] the medication should be given to the patient. Must be prefixed by 'ge'. Example: `ge2014-05-19T20:54:02.000Z`
+ `_lastUpdated`            | N         | [`date`]      | The date-time within which the most recent clinically relevant update was made to the medication. Must be prefixed by 'ge' or 'le'. Example: `ge2014-05-19T20:54:02.000Z`
  [`_count`]                | N         | [`number`]    | The maximum number of results to include in a page. Example: `50`
 
+ Notes:
+
+ - The `-timing-boundsPeriod` and `_lastUpdated` parameters may not be provided at the same time.
+ - When the `_lastUpdated` parameter is provided:
+   - Up to two dates-times may be provided.
```

Comment:
date-times

I would be clear here about how they can be provided (comma separated vs. two instances of the parameter)

Reply:
How about this:

```
- When using the `_lastUpdated` parameter:
   - It may be provided twice with each occurrence set to a single date-time.
   - Each date must have a time component and be prefixed by `ge` or `le`.
   - When provided twice, the earliest date-time must have the `ge` prefix and the latest date-time must have the `le` prefix.
```

Reply:
In a couple other resources, we do a single bullet instead of breaking it out

Observation/CarePlan:
The XXXXX parameter may be provided up to two times, and must use the ge or le prefixes. When provided twice, the lower value must have the ge prefix and the higher value must have the le prefix.

Procedure:
The XXXXX parameter must have a time, may be provided up to two times, and must use the ge or le prefixes. When provided twice, the lower value must have the ge prefix and the higher value must have the le prefix.

I think copying the language for we have for Procedure would be good